src/app-album-comments.component.js

```diff
@@ -0,0 +1,36 @@
+import template from './app-album-comments.html'
+
+export const AppAlbumCommentsComponent = {
+  controller : function ($state, SpotifyService) {
+
+    var ctrl = this;
```

Comment:
If you use a class, there is no need to catch the this, like on the angular legacy